tests/dbus/test_dbus.py

```diff
@@ -49,6 +50,7 @@ def setUp(self):
     eth0:
       dhcp4: true""")
         self.addCleanup(shutil.rmtree, self.tmp)
+        self.addCleanup(shutil.rmtree, '/tmp/netplan-config-BACKUP', True)
```

Comment:
Is this using the real /tmp during the tests? Would it make sense to also move this under /run or would that break something ?

Reply:
Yes. That's ugly. Let's move everything to `<TEST-ROOT>/run/netplan/config-*`, that way test can also be run in parallel.